projects/netgrif-components-core/src/lib/tabs/classes/opened-tab.ts

```diff
@@ -73,5 +97,7 @@ export class OpenedTab implements TabContent {
     public destroy(): void {
         this.tabSelected$.complete();
         this.tabClosed$.complete();
+        this._label$.text$.complete();
```

Comment:
it might be better to have a `complete()` method on the `_label$` object, that hides the internal implementation

Reply:
Done